spec/services/school_search_spec.rb

```diff
@@ -7,19 +7,19 @@
 
   describe "#call" do
     it "can search by urn" do
-      expect(described_class.call(query: school.urn)).to match([school])
+      expect(described_class.call(query: school.urn).schools).to match([school])
```

Comment:
it would be also good to have specs showing the value of `limit`, since that's being exposed from the service

Reply:
thanks I've added a test